Dido's project/StyleAbout.css

```diff
@@ -0,0 +1,87 @@
+body
+{
+    background-color: burlywood;
+}
+.H1
```

Comment:
Make sure you name your classes in CSS with the dashed case. You can check why [here](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/)